src/GutenbergCleanup/Utils.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace WPML\PB\GutenbergCleanup;
+
+use WPML\FP\Relation;
+
+class Utils {
+
+	/**
+	 * @param string $content
+	 *
+	 * @return bool
+	 */
+	public static function hasGutenbergMetaData( $content ) {
+		return (bool) preg_match( '/<!-- wp:/', $content );
+	}
+
+	/**
+	 * @param string $content
+	 *
+	 * @return string
+	 */
+	public static function stripHtmlComments( $content ) {
+		return preg_replace( '(<!--[^<]*-->)', '', $content );
```

Comment:
@strategio I've checked your regex. this will not remove comment like this:

```
<!-- wp:myblock block which will wrap all its content in <code> html tag -->
```

because of `<` inside of the comment block which is allowed, rare though

I tried and regex like `'/(<!--.*-->)/sUmg'` is working on multiline and handled `<` too.

Reply:
https://regex101.com/r/OEXpDN/1

Reply:
@kkarpieszuk, I don't think `<!-- wp:myblock block which will wrap all its content in <code> html tag -->` is a real case or maybe I didn't understand what you mean.

Anyway, I have strengthen the test constraint with your scenario and tests are still passing as you can see:

https://github.com/OnTheGoSystems/wpml-page-builders/pull/123/commits/e5246c72084360675213feaa3ea3df923f6ab31c